examples/ppo/utils_mujoco.py

```diff
@@ -0,0 +1,116 @@
+import gym
```

Comment:
Since you're using HalfCheeta-v4, we can use gymnasium no?
This version of halfcheetah with the gym from the CI makes it crash.
The CI uses gym 0.23 for D4RL compatibility, and gymnasium for newer stuff. 
(welcome to gym wonderland)

Reply:
I used HalfCheetaf-v4 in case the CI env did not have MuJoCo. The default config has HalfCheetah-v3. 
Should we use gymnasium instead of gym then?